src/AuthenticatedAPIClient/axiosConfig.js

```diff
@@ -68,7 +68,7 @@ function applyAxiosInterceptors(authenticatedAPIClient) {
     const isAuthUrl = authenticatedAPIClient.isAuthUrl(originalRequest.url);
     const accessToken = authenticatedAPIClient.getDecodedAccessToken();
     const tokenExpired = authenticatedAPIClient.isAccessTokenExpired(accessToken);
-    if (isAuthUrl || !tokenExpired) {
+    if (isAuthUrl || !accessToken || !tokenExpired) {
```

Comment:
How did this manifest itself?  Will there ever be a case where this code is called and no JWT is present?
Does `isAccessTokenExpired` handle NULL accessToken?

Reply:
This manifested itself in the tests of consuming apps. The apps' tests mock the API client and do not scaffold the request with a JWT.

It is possible that this code would be called when there is not JWT present. In practice this should not happen if the app is checking authentication status before rendering, but the app is in control of this.

`isAccessTokenExpired` considers a null accessToken to be expired.

Reply:
Got it. Thanks for the clarification.